patientsearch/src/js/components/PatientListTable.js

```diff
@@ -316,9 +316,9 @@ export default function PatientListTable() {
           method: "PUT",
           headers: {
             Accept: "application/json",
-            "Content-Type": "application/json",
-            body: rowData.resource ? JSON.stringify(rowData.resource) : null,
+            "Content-Type": "application/json"
           },
+          body: rowData.resource ? JSON.stringify(rowData.resource) : null
```

Comment:
YES! we should never be naming a `body` header.  Recipe for easy confusion.  Sorry if we had a broken API expecting that!  Should never be the case.